eth/main.cpp

```diff
@@ -962,10 +959,7 @@ int main(int argc, char** argv)
 	for (auto const& s: passwordsToNote)
 		keyManager.notePassword(s);
 
-	writeFile(configFile, rlpList(signingKey, author));
-
-	if (sessionKey)
-		signingKey = sessionKey;
+	writeFile(configFile, rlpList(author, author));
```

Comment:
Anyway this author is being read in the beginning from this file if written previously

And then passed to eth::Client::setAuthor (which sets it to some "preSeal" block)